apps/wallet/src/ui/app/pages/initialize/create/index.tsx

```diff
@@ -85,17 +85,19 @@ const CreatePage = () => {
                         <Button
                             type="submit"
                             disabled={!isValid || isSubmitting}
-                            mode="primary"
-                            size="large"
-                        >
-                            <Loading loading={isSubmitting}>
-                                Create Wallet
+                            size="tall"
+                            text={
```

Comment:
the `Button` component already has a `loading` prop IIRC, can we use that or is this a one-off loading UI? it would be nice if we could maintain consistency here

(I know you just tweaked the styles and didn't implement this haha)

Reply:
I'll take a look! I did notice that the loading wasn't actually working for some reason so I couldn't visually test it.